server/routerlicious/packages/lambdas-driver/src/kafka-service/checkpointManager.ts

```diff
@@ -94,6 +95,10 @@ export class CheckpointManager {
 	 */
 	public async flush(): Promise<void> {
 		if (this.lastCheckpoint) {
+            Lumberjack.info(`Checkpointing last recieved offset: ${this.lastCheckpoint.offset}`, {
+                offset: this.lastCheckpoint.offset,
+                partition: this.lastCheckpoint.partition,
+            })
```

Comment:
Hey Xin..I wanted to understand your comment about the frequency of this log..this should ideally happen only when the service stops..that shouldn't be too often right?

Reply:
Hi @zhangxin511, I added these logs at @pradeepvairamani's request, because similar logs were helpful in debugging our checkpoints. Like he mentioned, this log should only be created on a service stop/restart. If we decide to keep this log, I can add the documentId and tenantId.

Reply:
I am okay with removing this log too if that complicates this PR. We can do this in a different PR.

Reply:
Sure, as long as this when service stop, this is good, thx for confirming